QEfficient/customop/ctx_scatter_gather.py

```diff
@@ -79,3 +79,84 @@ def setup_context(ctx, inputs, outputs):
     @staticmethod
     def symbolic(g: torch.Graph, data: torch.Value, ctx_indices: torch.Value) -> torch.Value:
         return g.onnxscript_op(CtxGather, data, ctx_indices).setTypeAs(data)
+
+
+@onnxscript.script(onnxscript.values.Opset("com.qualcomm.cloud", 1))
```

Comment:
remove?

Reply:
Didnt get you. Additional space? Will remove CB specific CtxGatherFunc and CtxScatterFunc from stx_scatter_gather.py as its maintained in different file.